src/test/java/com/github/themeetgroup/kafka/connect/rabbitmq/sink/docker/rabbitmq/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rabbitmq:3.7.10-management
```

Comment:
In Java & Scala we put non-source like this in resources, can be moved to `/src/test/resources/Dockerfile`